### PR TITLE
Database/procedures

### PR DESCRIPTION
**Pull request by RaresMrc**
Adding almost all procedures from the third module together with a few utility functions.

### CODE REVIEW

**Review by RaresMrc — COMMENTED** [review]
### I'm just **testing** the feature, **don't mind** this comment.

File: database/procedures/get_appointed_patients.sql
Comment (bmarcu13) [inline database/procedures/get_appointed_patients.sql:1]: Procedures should be kept in subdirectories representing the functionality module they belong to. Please create the needed subdirectory(ies) and move them accordingly.

File: database/procedures/get_medic_name.sql
Comment (bmarcu13) [inline database/procedures/get_medic_name.sql:1]: We should keep helper functions for procedures in a separate folder. e.g. `database/helper_functions`

**Review by bmarcu13 — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Review by bmarcu13 — APPROVED** [review]

**Pull request merged**